windows/NsisInclude/FilesToInstall.nsh

```diff
@@ -0,0 +1,255 @@
+SetOutPath "$INSTDIR\share\glib-2.0"
+SetOutPath "$INSTDIR\share\glib-2.0\schemas"
+File "..\dist\share\glib-2.0\schemas\gschemas.compiled"
+SetOutPath "$INSTDIR\share\icons"
+SetOutPath "$INSTDIR\share\icons\Adwaita"
+File "..\dist\share\icons\Adwaita\index.theme"
+SetOutPath "$INSTDIR\share\icons\Adwaita\16x16"
+SetOutPath "$INSTDIR\share\icons\Adwaita\16x16\actions"
+File "..\dist\share\icons\Adwaita\16x16\actions\edit-clear-all.png" "..\dist\share\icons\Adwaita\16x16\actions\edit-delete.png" "..\dist\share\icons\Adwaita\16x16\actions\edit-find.png" "..\dist\share\icons\Adwaita\16x16\actions\process-stop.png"
+SetOutPath "$INSTDIR\share\icons\Adwaita\24x24"
+SetOutPath "$INSTDIR\share\icons\Adwaita\24x24\actions"
+File "..\dist\share\icons\Adwaita\24x24\actions\edit-delete.png" "..\dist\share\icons\Adwaita\24x24\actions\edit-find.png" "..\dist\share\icons\Adwaita\24x24\actions\process-stop.png"
+SetOutPath "$INSTDIR\share\icons\Adwaita\32x32"
+SetOutPath "$INSTDIR\share\icons\Adwaita\32x32\actions"
+File "..\dist\share\icons\Adwaita\32x32\actions\edit-clear-all.png" "..\dist\share\icons\Adwaita\32x32\actions\edit-delete.png" "..\dist\share\icons\Adwaita\32x32\actions\edit-find.png" "..\dist\share\icons\Adwaita\32x32\actions\process-stop.png"
+SetOutPath "$INSTDIR\share\icons\Adwaita\scalable"
+SetOutPath "$INSTDIR\share\icons\Adwaita\scalable\actions"
+File "..\dist\share\icons\Adwaita\scalable\actions\list-add-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\list-remove-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\pan-down-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\pan-end-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\window-close-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\window-maximize-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\window-minimize-symbolic.svg" "..\dist\share\icons\Adwaita\scalable\actions\window-restore-symbolic.svg"
+SetOutPath "$INSTDIR\share\icons\hicolor"
+File "..\dist\share\icons\hicolor\index.theme"
+SetOutPath "$INSTDIR\share\locale"
+SetOutPath "$INSTDIR\share\locale\af"
```

Comment:
The `locale` folder should not be installed for the English-only version of the installer.